liquibase-couchbase/src/test/java/liquibase/ext/couchbase/database/CouchbaseConnectionTest.java

```diff
@@ -59,6 +64,25 @@ void Should_open_connection_correctly() {
         assertThat(connection.getDatabaseMinorVersion()).isEqualTo(0);
     }
 
+    @Test
+    @SneakyThrows
+    void Should_open_connection_on_bucket() {
+        try (MockedStatic<Cluster> mockedStatic = Mockito.mockStatic(Cluster.class)) {
+            String bucketName = "bucket";
+
+            mockedStatic.when(() -> Cluster.connect(anyString(), any())).thenReturn(cluster);
+
+            Properties driverProperties = new Properties();
```

Comment:
`driverProperties` creating with user and password could be extracted to a private method.

Reply:
done